includes/api/class-edd-api.php

```diff
@@ -163,6 +209,65 @@ public function query_vars( $vars ) {
 	}
 
 	/**
+	 * Sets the version of the API that was queried.
+	 *
+	 * Falls back to the default version if no version is specified
+	 *
+	 * @access private
+	 * @since 2.4
+	 */
+	private function set_queried_version() {
+
+		global $wp_query;
+
+		$version = $wp_query->query_vars['edd-api'];
+
+		if( strpos( $version, '/' ) ) {
+
+			$version = explode( '/', $version );
+			$version = strtolower( $version[0] );
+
+			$wp_query->query_vars['edd-api'] = str_replace( $version . '/', '', $wp_query->query_vars['edd-api'] );
+
+			if( array_key_exists( $version, $this->versions ) ) {
+
+				$this->queried_version = $version;
+
+			} else {
+
+				$this->queried_version = $this->get_default_version();
+
+			}
+
+		} else {
+
+			$this->queried_version = $this->get_default_version();
+
+		}
+
+	}
+
+	/**
+	 * Retrieves the default version of the API to use
+	 *
+	 * @access private
+	 * @since 2.4
+	 * @return string
+	 */
+	private function get_default_version() {
+
+		$version = get_option( 'edd_default_api_version' );
+
+		if( defined( 'EDD_API_VERSION' ) ) {
+			$version = EDD_API_VERSION;
+		} elseif( ! $version ) {
+			$version = 'v1';
```

Comment:
this should be 1, not v1 though right? We should define the version as a number, even if we display it as v{num}